appveyor.yml

```diff
@@ -91,21 +91,13 @@ test_script:
       --extra-lib-dirs="C:\xz_extracted\bin_x86-64"
       --extra-include-dirs="%WORK_DIR%\rocksdb\include"
       --extra-lib-dirs="%WORK_DIR%"
-  - stack exec -- cardano-wallet-hs2purs
-  # Prepare files for frontend build
+  # Cardano pieces, modulo the frontend
+  - mkdir daedalus
   - copy log-config-prod.yaml daedalus\
   - copy lib\configuration.yaml daedalus\
   - copy lib\*genesis*.json daedalus\
   - copy cardano-launcher.exe daedalus\
-  - copy cardano-node.exe daedalus\
-  # Build the bridge
-  - cd daedalus
-  - Echo %APPVEYOR_BUILD_VERSION% > build-id
-  - Echo %APPVEYOR_REPO_COMMIT% > commit-id
-  - Echo https://ci.appveyor.com/project/%APPVEYOR_ACCOUNT_NAME%/%APPVEYOR_PROJECT_SLUG%/build/%APPVEYOR_BUILD_VERSION% > ci-url
-  - ps: Install-Product node 7
-  - ..\scripts\ci\appveyor-retry call npm install
-  - npm run build:prod
+  - copy cardano-node.exe daedalus
```

Comment:
I really cannot judge if the following is correct. Maybe let's not bother now and when we open a PR from `feature/new-wallet` into `develop` we ask devops to review everything.